target_gsheet.py

```diff
@@ -140,12 +145,16 @@ def persist_lines(service, spreadsheet, lines):
             schema = schemas[msg.stream]
             validate(msg.record, schema)
             flattened_record = flatten(msg.record)
-            
+
             matching_sheet = [s for s in spreadsheet['sheets'] if s['properties']['title'] == msg.stream]
```

Comment:
one thing I don't like about those singer target is they don't allow us to overwrite the sheet(the tab in a google sheet) and relying on title matching of stream only. can you add a `sheet_title` or sth like that in config.json so we can force it to the exact sheet we want?

Reply:
Good catch. Done